bluepyopt/deapext/utils.py

```diff
@@ -83,6 +83,23 @@ def __init__(self, *args, **kwargs):
         super(WSListIndividual, self).__init__(*args, **kwargs)
 
 
+def generate_starters(ind_size, lambda_, clip=True, clip_threshold=0.8):
+    """Generate a new set of starter individual uniformaly distributed in a
+    hypercube centered at zero and of length 2. * clip_threshold. Used by SO
+    and MO-CMA"""
+
+    ubound = clip_threshold if clip else 1.
+    lbound = - ubound
+
+    starters = [
+        WSListIndividual(
+            numpy.random.uniform(lbound, ubound, ind_size), obj_size=ind_size)
+        for i in range(lambda_)
+    ]
+
+    return starters
+
```

Comment:
Overall what's the reason for changing the initialisation? Did the overexploring of the bounds lead to a bad exploration of the overall space? Or do we expect the bounds to not be good parameter values in general?

Reply:
Both. The bounds are being over-explored because the parameter sets are brought back to the bounds when they are spawned outside.
And generally, our bounds are a bit "extreme" compare to the expected values and thus the solution is likely not on the bounds (but most of the models that crash are there).

Reply:
Note that, here if the god solution is in the missing 20%, we do not prevent the centroid to go there during the optimization.